content/pt-br/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -32,10 +32,10 @@ Existem os seguintes métodos para instalar o kubectl no Linux:
    {{< note >}}
 Para fazer o download de uma versão específica, substitua a parte `$(curl -L -s https://dl.k8s.io/release/stable.txt)` do comando pela versão específica.
 
-Por exemplo, para fazer download da versão {{< param "fullversion" >}} no Linux, digite:
+Por exemplo, para fazer download da versão v{{< skew currentPatchVersion >}} no Linux, digite:
```

Comment:
(nit)
```suggestion
Por exemplo, para fazer download da versão {{< skew currentPatchVersion >}} no Linux, digite:
```
“versão” already covers this (IMO)

Reply:
True. I added the "v" mainly to keep the previous style.